hack/shred_accounts.sh

```diff
@@ -0,0 +1,185 @@
+#!/bin/bash
+
```

Comment:
Haven't checked with this script yet, but in general, it's a good idead to add 
```
set -euo pipefail
```
at the top of your bash script to prevent hard to debug errors in your scripts. See [here](https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/) for example on what exactly that does.

Reply:
set all but "-u", apparently "-u" is kind of sensitive, it was throwing an error about unbound variable "$1" in functions like:
```
function foo {
  echo "$1"
}
```
Tried a couple things but wasnt able to find a quick workaround on google.